Sources/Core/ServerConfiguration.swift

```diff
@@ -77,6 +80,69 @@ struct ServerConfiguration: ServerConfigurationProviding {
     return remotes.compactMap { DialogFlow(remote: $0) }
   }
 
+  // TODO: Revisit the need for a default since all it really provides
+  // are dialog flows which are already available as a static property
+  /**
+ This initializer should only use this for providing a default while
+   an up-to-date version is being fetched from the server
+   */
+  init(
```

Comment:
wao, long init :D

Reply:
Yeah. The number of parameters and the amount of relatively unstructured information in this config is a clear code-smell imho. I think it would be a good goal for post v1 to structure these into more discrete objects.